src/videotilecontroller/VideoTileController.ts

```diff
@@ -23,10 +23,12 @@ export default interface VideoTileController {
   /**
    * Unbinds the video element from the tile if it exists for the provided tileId.
    * The video tile's bounded video element and that element's width and height are set to null.
+   * @param cleanUpVideoElement By default, the bounded video element's srcObject is also set to null using [[disconnectVideoStreamFromVideoElement]].
```

Comment:
nit: I don't think you need to mention a non-public method: `disconnectVideoStreamFromVideoElement`.

Reply:
Makes sense, removed and rephrased 🙏